numpy/core/_exceptions.py

```diff
@@ -88,13 +88,24 @@ def __init__(self, ufunc, casting, from_, to, i):
         self.in_i = i
 
     def __str__(self):
-        # only show the number if more than one input exists
-        i_str = "{} ".format(self.in_i) if self.ufunc.nin != 1 else ""
+        if self.ufunc.nout > 1:
+            from_outs = ", ".join(["out{}={}".format(i+1, f.name) for i, f in enumerate(self.from_) if i>=self.ufunc.nin])
+            to_outs = ", ".join(["out{}={}".format(i+1, t.name) for i, t in enumerate(self.to) if i>=self.ufunc.nin])
```

Comment:
To be clear - this `out{}=` needs changing to match the way you updated it for the other one.

Reply:
Thanks, coming back into it I had forgotten there were two places where this applied.